examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,965 @@
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = torch.tensor([1,2,3,4,5])\n",
+    "y = torch.tensor([1,1,1,1,1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "और अब - चलो हमारे Tensor को बॉब(Bob) को भेजें !!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x_ptr = x.send(bob)\n",
+    "y_ptr = y.send(bob)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(Wrapper)>[PointerTensor | me:8030612864 -> bob:14597461368]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x_ptr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "BOOM! अब बॉब(Bob) के दो टेनर्स हैं! मुझे विश्वास नहीं है? अपने लिए इसे देख लें!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{14597461368: tensor([1, 2, 3, 4, 5]), 86863779266: tensor([1, 1, 1, 1, 1])}"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bob._objects"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "z = x_ptr + x_ptr"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(Wrapper)>[PointerTensor | me:25749208299 -> bob:51980816025]"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "z"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{14597461368: tensor([1, 2, 3, 4, 5]),\n",
+       " 86863779266: tensor([1, 1, 1, 1, 1]),\n",
+       " 51980816025: tensor([ 2,  4,  6,  8, 10])}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bob._objects"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "अब कुछ नोटिस करो। जब हमने `x.send (bob)` को फोन किया तो उसने एक नई वस्तु लौटा दी जिसे हमने` x_ptr` कहा। यह एक टेंसर के लिए हमारा पहला  पॉइंटर  है। दसियों को इंगित करने वाले वास्तव में स्वयं डेटा नहीं रखते हैं। इसके बजाय, वे बस एक मशीन पर संग्रहीत एक डेटा (डेटा के साथ) के बारे में मेटाडेटा होते हैं। इन टेंसरों का उद्देश्य हमें एक सहज ज्ञान युक्त एपीआई देना है जो अन्य मशीन को इस टेंसर का उपयोग करके कार्यों की गणना करने के लिए बताए। आइए मेटाडेटा पर एक नज़र डालें जिसमें संकेत होते हैं।"
```

Comment:
Spacing issue with x_ptr need to swap out space in backticks to the left. I think data outside parenthesis here(डेटा (डेटा के साथ)) should be tensor instead. What do you think?

Reply:
Also in the last line here word pointers is replaced with संकेत need to fix that for it to make sense